notes/stv/20200408-Spark-on-K8s.txt

```diff
@@ -288,28 +288,6 @@ Key name [stv-master]
 | flavor_id           | 20061eba-9e88-494c-95a3-41ed77721244                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 +---------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-## [Admin Node]  Create storage Class
```

Comment:
Does this mean the nodes don't have any storage on them ?

Reply:
I noticed that the Storage class create command is run a few lines after, so this was probably a mistake